File-Formats/DocIO/Find-item-in-word-document.md

```diff
@@ -0,0 +1,738 @@
+        outputStream.Position = 0;
+        //Download as a Word document in the browser.
+        return File(outputStream, "application/msword", "Sample.docx");
+    }
+}
+{% endhighlight %} 
+
+{% highlight c# tabtitle="Xamarin" %}
+//Open the file as Stream.
+using (Stream docStream = typeof(App).GetTypeInfo().Assembly.GetManifestResourceStream("Sample.Assets.Input.docx"))
+{
+    //Load the file stream into a Word document.
+    using (WordDocument document = new WordDocument(docStream, FormatType.Docx))
+    {
+        //Find all footnote and endnote by EntityType in Word document.
+        List<Entity> footNotes = document.FindAllItemsByProperty(EntityType.Footnote, null, null);
+        //Iterates the footnotes and remove the footnotes.
+        for (int i = 0; i < footNotes.Count; i++)
+        {
+            WFootnote footnote = footNotes[i] as WFootnote;
+            footnote.OwnerParagraph.ChildEntities.Remove(footnote);
+        }
+
+        //Find all fields by FieldType.
+        List<Entity> fields = document.FindAllItemsByProperty(EntityType.Field, "FieldType",FieldType.FieldHyperlink.ToString());
+        //Iterates the fields
+        for (int i = 0; i < fields.Count; i++)
+        {
+            //Creates hyperlink instance from field to manipulate the hyperlink.
+            Hyperlink hyperlink = new Hyperlink(fields[i] as WField);
+            //Modifies the Uri of the hyperlink.
+            if (hyperlink.Type == HyperlinkType.WebLink && hyperlink.TextToDisplay == "HTML")
+                hyperlink.Uri = "http://www.w3schools.com/";
+        }
+        //Save a Word document to the MemoryStream.
+        MemoryStream outputStream = new MemoryStream();
+        document.Save(outputStream, FormatType.Docx);
+        //Save the stream as a file in the device and invoke it for viewing. 
+        Xamarin.Forms.DependencyService.Get<ISave>().SaveAndView("Sample.docx", "application/msword", outputStream);
+    }
+}
+//Download the helper files from the following link to save the stream as file and open the file for viewing in Xamarin platform
+//https://help.syncfusion.com/file-formats/docio/create-word-document-in-xamarin#helper-files-for-xamarin
+{% endhighlight %} 
+
+{% endtabs %}
+
+## Find all items by properties
+
+Using FindAllItemsByProperties API, find all the items in the Word document based on multiple property names and their corresponding values.
+
+The following code example illustrates how to find all the items in Word document based on multiple property names and their corresponding values.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+//Load an existing Word document.
+using (WordDocument document = new WordDocument("Input.docx", FormatType.Docx))
+{
+    string[] propertyNames = { "ContentControlProperties.Title", "ContentControlProperties.Tag" };
+    string[] propertyValues = { "CompanyName", "CompanyName" };
+
+    //Find all block content control by Title and Tag. 
```

Comment:
//Find all block content controls by Title and Tag.

Reply:
Addressed